.versions

```diff
@@ -0,0 +1,4 @@
+gilbertwat:moment-recur@1.0.4
+meteor@1.1.6
+momentjs:moment@2.10.3
+underscore@1.0.3
```

Comment:
I'm troubled to have multiple versions of dependency listings.  Can this file get auto-generated from the gulpfile & bower.json?